docs(gen): docs content review (#2759)
scaleway/docs-content: containers/kubernetes/how-to/deploy-ingress-controller.mdx

  - You have [created](/containers/kubernetes/how-to/create-cluster/) a Kubernetes Kapsule cluster
</Message>

1. Navigate to the **Containers** section within the [Scaleway console](https://console.scaleway.com) sidebar and select **Kubernetes**. This action will direct you to the Kubernetes Kapsule dashboard.
2. Locate the cluster you want to modify and click on its name to access the **cluster dashboard**.
3. Within the dashboard, navigate to the **Easy Deploy** tab to display the Easy Deploy feature. Then click **Deploy application** to create a new application.
4. Select the **Application Library** application type. Then select the desired ingress controller for deployment onto your cluster.
   <Message type="note">

>      You have the option to customize the Helm chart to suit your needs, or you can retain the default configuration. Helm charts are formatted in YAML. Keep in mind that any alterations carry inherent risks and may impact the successful deployment of your application.

Reviewer: ```suggestion
     You can customize the Helm chart or keep its default configuration. Helm charts are formatted in YAML. Keep in mind that any alterations carry inherent risks and may impact the successful deployment of your application.
```
